### PR TITLE
Text utilities & lists

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image

### CODE REVIEW

File: components/Badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    justify-content: center;
+    gap: 25px;
+}
+.text__badge{
+    position: relative;
+    width: 160px;
+    height: 42px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (rohanmathur91) [inline components/Badges/badges.css:23]: You can have a utility class for flex

File: components/Badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    top: -10px;
+    left: 34px;
+    background-color: var(--badge-bg-color);
+    padding: 6px;
+    border-radius: 50%;
+    border: 3px solid white;
+}
+button{
+    padding: 12px 22px;
+}
```
Comment (rohanmathur91) [inline components/Badges/badges.css:54]: use rem instead of px

File: components/Buttons/button.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+*/
+.lg-btn{
+    font-size: 1.1rem;
+    padding: 0.7rem 1.5rem;
+    min-width: 80px;
+}
+.sm-btn{
+    font-size: 0.9rem;
+    padding: 0.3rem 0.7rem;
+    width: 112px;
```
Comment (rohanmathur91) [inline components/Buttons/button.css:41]: Same as above, use rem

File: components/Buttons/button.css [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+}
+/* 
+CSS FOR TOGGLE BTN
+*/
+.btn-toggle{
+    width: 26px;
+    height: 44px;
```
Comment (rohanmathur91) [inline components/Buttons/button.css:48]: Same as above

File: components/Buttons/button.css [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+    width: 0;
+    margin: 0;
+    opacity: 0;
+}
+.btn-toggle .btn-slider{
+    position: absolute;
+    top:0;
+    left:9px;
+    height: 26px;
+    width: 26px;
```
Comment (rohanmathur91) [inline components/Buttons/button.css:65]: same as above

File: components/Input/input.css [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.radio input,.checkbox input{
+    cursor: pointer;
+    margin: 0 10px;
+}
+
+.radio.disabled label,.checkbox.disabled label{
+    color:#888;
+}
```
Comment (rohanmathur91) [inline components/Input/input.css:69]: Use color variable for this

**Review by rohanmathur91 — CHANGES_REQUESTED** [review]
In a lot of places you have used px, so try to use rem.

File: components/Lists/list.html [hunk 7]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (surbhi84) [inline components/Lists/list.html:6]: suggestion: you can add author and description meta tags as well.

File: components/Text Utilities/text.html [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+    <link rel="stylesheet" href="text.css">
+</head>
+<body>
+    <h1>H1 - Heading 1 </h1>
+    <h2>H2 - Heading 2 </h2>
+    <h3>H3 - Heading 3 </h3>
+    <h4>H4 - Heading 4 </h4>
+    <h5>H5 - Heading 5 </h5>
+    <h6>H6 - Heading 6 </h6>
+
```
Comment (surbhi84) [inline components/Text Utilities/text.html:17]: why headings ?

**Review by surbhi84 — APPROVED** [review]
looks good to me.

**Review by Jay-Gadhiya — APPROVED** [review]

**Pull request merged**